apkid/rules/dex/packers.yara

```diff
@@ -243,3 +243,21 @@ rule jar_pack01 : packer
     is_dex and
     ($pre_jar and $jar_data and $post_jar)
 }
+
+rule appsuit_packer : packer
```

Comment:
Is this a different product / configuration from the appsuit obfuscator? If it's the same thing, then it's more accurate to call one of the rules an obfuscator and a packer and just use one rule. If only one rule is used, my money is on a dex rule, since it's fingerprinting the code. If you think it's useful / necessary, you could also have a silent apk rule that the dex rule depends on.

Reply:
It's different configuration indeed. There are two rules called `appsuit_obfuscator` and `appsuit_packer` in their responsible source files. What's wrong?

Reply:
Ahh, all is clear now. Different configurations can certainly be different rules / variants. From what you just told me, I suggest changing the rule names and descriptions to include "obfuscator" or "packer"